packages/dd-trace/src/scope.js

```diff
@@ -34,6 +34,15 @@ class Scope {
     }
   }
 
+  getDataStreamsContext () {
```

Comment:
These should not go on the scope manager as it's meant only for span management.

Reply:
I extended `scope` to be the full context, not only span scope, but also data streams scope.
Where would you see that data streams context live instead?

Reply:
Moved outside of scope.